src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -99,7 +99,7 @@ HighlightingPage::HighlightingPage()
                         .getElement();
 
                 view->addRegexHelpLink();
-                view->getTableView()->horizontalHeader()->hideSection(4);
+                view->getTableView()->horizontalHeader()->hideSection(5);
```

Comment:
```suggestion
                view->getTableView()->horizontalHeader()->hideSection(
                    HighlightModel::Column::UseRegex);
                view->getTableView()->horizontalHeader()->hideSection(
                    HighlightModel::Column::CaseSensitive);
```

Your change gets rid of the "Case-sensitive" column but the "Enable regex" column is then shown instead (se image)
.